docs: improving getting started docs for `mainnet` (#3500)
FuelLabs/fuels-ts: apps/docs/src/guide/getting-started/index.md

# Getting Started

Welcome to `fuels` Typescript SDK!

> We prepared a couple guides to walk you through your first steps:

Reviewer: ```suggestion
We prepared some guides to walk you through your first steps:
```
